assets/js/neoforge.js

```diff
@@ -9,14 +9,24 @@ async function loadLatestVersions(minecraftVersions) {
 	let gav;
 	let fn;
 	let mcvers;
+    let note;
+    let dropDown_VAL;
+    let badges_beta;
+    let badges_new;
 	if (mcVersion.startsWith("1.20.1")) {
 		gav = LEGACY_GAV;
-                fn = "forge";
-		mcvers = "1.20.1"
+        fn = "forge";
+		mcvers = "1.20.1";
+        badges_new = "";
+        badges_beta = "";
+        dropDown_VAL = "";
 	} else {
 		gav = FORGE_GAV;
-                fn = "neoforge";
-		mcvers = `1.${mcVersion}`
+        fn = "neoforge";
+		mcvers = `1.${mcVersion}`;
+        badges_beta = "";
+        badges_new = `<font class="badges_new">NEW</font>`;
```

Comment:
We probably only want this for the latest release? (assuming we fetch that info automatically, see other comment)

Reply:
Ah, yes! Now it's only for a single version; the latest one.